manager/pom.xml

```diff
@@ -106,6 +106,13 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-thymeleaf</artifactId>
         </dependency>
+
+        <!--webflux-->
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-webflux</artifactId>
+        </dependency>
+
```

Comment:
hi, due we add new thrid denpedency, we need update the license doc here. https://github.com/apache/hertzbeat/tree/master/material/licenses